### ISSUE TITLE
The images in App Store newer than here

### ISSUE THREAD

**Issue by tobias38fmsw** [keepassium/KeePassium#61]
You can take your screenshots (some) from App Store. And update here on github page.

**Comment by tobias38fmsw** [comment]
would be best with iOS 13 and current KeePassium version.

I think every year should be updated with current iOS/iPadOS and KeePassium version. 😀

**Comment by keepassium** [comment]
Good point! And now that there are 6 languages, I’d better automate the process...

**Comment by keepassium** [comment]
Arrgh... I cannot get iOS 13 simulator to work properly with files. So the dark mode screenshots are stuck until I (or Apple?) figure this out...
